invenio_github/tasks.py

```diff
@@ -137,14 +137,15 @@ def process_release(release_id, verify_sender=False, use_extra_metadata=True):
         raise InvalidSenderError(
             event=release.event.id, user=release.event.user_id)
 
+    matched_error_cls = None
```

Comment:
I would also move the `matched_ex` here just for consistency and to keep things together